src/internal/operators/switchMap.ts

```diff
@@ -36,6 +36,16 @@ export function switchMap<T, R, O extends ObservableInput<any>>(project: (value:
  * subsequent inner Observables.
  *
  * ## Example
+ * Generate new Observable according to source Observable values
+ * ```javascript
+ * import { of } from 'rxjs';
+ * import { switchMap } from 'rxjs/operators';
+ *
+ * const switched = of(1,2,3).pipe(switchMap((x: number) => of( x, x ** 2, x ** 3)));
+ * switched.subscribe(x => console.log(x));
+ * ```
+ *
```

Comment:
Have you considered adding what the result would look like? That'd be quite valuable. It can be in the format:

```
Outputs
1
1
1
2
4
8
... and so on
```

Reply:
added the outputs in new commit